lib/counter_culture/collection.rb

```diff
@@ -0,0 +1,72 @@
+module CounterCulture
+  class Collection
+    include Enumerable
+
+    def initialize(arr=Array.new)
+      @arr = arr
+    end
+
+    def fix_counts(options={})
```

Comment:
I don't have any rigid rules regarding this. I just look at the problem at hand and try and find the best solution. I continue to think the domain of this code is closer to the concept of the `Reconciler` than to that of a `Collection`. So I believe it belongs there.